test/parallel/test-stream-pump.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const stream = require('stream');
+require('../common');
+const assert = require('assert');
+const crypto = require('crypto');
+const pump = stream.pump;
+// tiny node-tap lookalike.
```

Comment:
I’m really not keen on introducing more of these in the tests, the ones that use “tiny node-tap lookalike” are usually far from readable…